social/src/main/java/org/exoplatform/platform/qa/ui/social/pageobject/AddUsers.java

```diff
@@ -27,17 +34,37 @@ public AddUsers(TestBase testBase) {
    * @param Firstname
    * @param lastName
    */
-  public void addUser(String userName, String Password, String email, String Firstname, String lastName) {
+  public void addUser(String userName, String password, String email, String firstname, String lastName) {
     info("Add an user");
-    evt.type(ELEMENT_USERNAME, userName, true);
-    evt.type(ELEMENT_PASSWORD, Password, true);
-    evt.type(ELEMENT_CONFIRM_PASSWORD, Password, true);
-    evt.type(ELEMENT_EMAIL, email, true);
-    evt.type(ELEMENT_FIRSTNAME, Firstname, true);
-    evt.type(ELEMENT_LASTNAME, lastName, true);
-    evt.click(ELEMENT_SAVE);
-    Utils.pause(2500);
-    evt.click(ELEMENT_CONFIRM_INFORMATION);
+    $(ELEMENT_USERNAME).waitUntil(appears, Configuration.timeout);
+    $(ELEMENT_USERNAME).setValue(userName);
+    $(ELEMENT_PASSWORD).setValue(password);
+    $(ELEMENT_CONFIRM_PASSWORD).setValue(password);
+    $(ELEMENT_EMAIL).setValue(email);
+    $(ELEMENT_FIRSTNAME).setValue(firstname);
+    $(ELEMENT_LASTNAME).setValue(lastName);
+    $(ELEMENT_SAVE).click();
+    $(byText("You have registered a new account.")).waitUntil(appears, Configuration.timeout);
+    $(byText("OK")).click();
```

Comment:
Please use a variable in SocialLocator instead of using string, to be compliant with other tests.